package.json

```diff
@@ -3,34 +3,52 @@
   "version": "0.0.0",
   "scripts": {
     "ng": "ng",
-    "start": "ng serve",
+    "start": "node server.js",
     "build": "ng build",
     "test": "ng test",
```

Comment:
can you find a way to trim these dependencies? I bet you're not using most of them.